src/VirtualMonoRepo/baseline-file.txt

```diff
@@ -0,0 +1,23 @@
+*.png
```

Comment:
Consider renaming this file to have a more descriptive name - e.g. `allowed-binaries.txt`, `allowed-binaries-baseline.txt`, `binaries-baseline.txt`

Reply:
renamed to `allowed-binaries.txt`